src/main.ts

```diff
@@ -37,9 +42,23 @@ async function bootstrap(): Promise<void> {
   const options = new DocumentBuilder()
     .setTitle('Polymesh REST API')
     .setDescription('RESTful access to the Polymesh blockchain')
-    .setVersion('1.0')
-    .build();
-  const document = SwaggerModule.createDocument(app, options);
+    .setVersion('1.0');
+
+  const authStrategies = parseAuthStrategyConfig(process.env.AUTH_STRATEGY || '');
```

Comment:
I thought of this now:

I think you can use a method like `app.get<ConfigService>(ConfigService)` and use that as if it was injected.

We use the syntax in tests. I think it slightly cleaner than reading `env` here directly and should be more consistent.